rainbowsaddle/__init__.py

```diff
@@ -164,6 +177,8 @@ def main():
             'rainbow-saddle PID')
     parser.add_argument('--gunicorn-pidfile', help='a filename to store the '
             'gunicorn PID')
+    parser.add_argument('--statsd', help='Set it to true, if you dont set it by default its false')
```

Comment:
Please change the help text to be more descriptive. Perhaps "Enable statsd metrics"?